components/camel-aws/camel-aws2-kinesis/src/main/java/org/apache/camel/component/aws2/kinesis/client/KinesisClientFactory.java

```diff
@@ -44,4 +45,14 @@ public static KinesisInternalClient getKinesisClient(Kinesis2Configuration confi
             return new KinesisClientStandardImpl(configuration);
         }
     }
+
+    /**
+     * Return the standard aws Kinesis Async client.
+     *
+     * @param  configuration configuration
+     * @return               KinesisAsyncClient
+     */
+    public static KinesisAsyncInternalClient getKinesisAsyncClient(Kinesis2Configuration configuration) {
```

Comment:
The default and profile credentials are missing. It should be the same as the Sync client, so you should provide the other two implementations

Reply:
It's done